core/templates/components/question-directives/question-player/skill-mastery-modal.controller.spec.ts

```diff
@@ -44,14 +44,16 @@ describe('Skill Mastery Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
+  it('should initialize correctly $scope properties after controller' +
+    ' initialization', function() {
     expect($scope.userIsLoggedIn).toEqual(userIsLoggedIn);
     expect($scope.skillId).toEqual(skillId);
     expect($scope.masteryChange).toEqual('mastery change');
   });
 
-  it('should open concept card', function() {
-    $scope.openConceptCardModal();
-    expect(openConceptCardModal).toHaveBeenCalledWith([skillId]);
-  });
+  it('should open concept card when clicking button to open concept card',
```

Comment:
Somewhere in the name include that skillId should be passed into the modal

Reply:
Done.